scout/utils/__init__.py

```diff
@@ -1 +1,17 @@
 # -*- coding: utf-8 -*-
+import logging
+
+LOG = logging.getLogger(__name__)
+
+
+def call_safe(fun, unknown_var):
```

Comment:
Same as in the other PR, I think it would be best to move this function to be among other functions that do similar things (/scout/utils/convert.py)

Reply:
Moved!